tools/package.json

```diff
@@ -0,0 +1 @@
+^0.0.1
```

Comment:
Why this file is modified? All files that suppose to be modified should be under `servers` and script will be used to modify when we switch to `*publish` branch. Other branches will continue having the file reference of the package.